dev/github/run_tests.sh

```diff
@@ -30,6 +30,16 @@ if [ -n "${ALLUXIO_MVN_PROJECT_LIST}" ]; then
   mvn_args+="-am -pl ${ALLUXIO_MVN_PROJECT_LIST}"
 fi
 
+mvn_compile_args=""
+if [ -n "${ALLUXIO_MVN_PROJECT_LIST_COMPILE}" ]; then
+  mvn_compile_args+="-am -pl ${ALLUXIO_MVN_PROJECT_LIST_COMPILE}"
+fi
+
+mvn_test_args=""
+if [ -n "${ALLUXIO_MVN_PROJECT_LIST_TEST}" ]; then
+  mvn_test_args+="-am -pl ${ALLUXIO_MVN_PROJECT_LIST_TEST}"
+fi
```

Comment:
`mvn _tests_args` should be defined immediately before it is first used, above `# Run tests`. this helps localize the variable usage so that it is accessible only as it is needed